aip/0142.md

```diff
@@ -134,7 +134,7 @@ use.
 <!-- prettier-ignore-start -->
 [aip-213]: ./0213.md
 [date]: https://github.com/googleapis/api-common-protos/blob/master/google/type/date.proto
-[datetime]: https://github.com/googleapis/api-common-protos/blob/master/google/type/datetime.proto
+[datetime]: https://github.com/googleapis/googleapis/blob/master/google/type/datetime.proto
```

Comment:
Note: I'm not certain where the canonical location for this file (or any of the others in the PR) is supposed to be; this is just a guess based on a quick search.